plugins/catalog-backend/src/ingestion/processors/gitlab/client.ts

```diff
@@ -20,14 +20,22 @@ import {
   GitLabIntegrationConfig,
 } from '@backstage/integration';
 import { Logger } from 'winston';
+import { BlobsSearchResult, GitLabProject } from './types';
+import { CacheClient } from '@backstage/backend-common';
 
 export class GitLabClient {
   private readonly config: GitLabIntegrationConfig;
   private readonly logger: Logger;
+  private readonly cache: CacheClient | undefined;
 
-  constructor(options: { config: GitLabIntegrationConfig; logger: Logger }) {
+  constructor(options: {
+    config: GitLabIntegrationConfig;
+    logger: Logger;
+    cache?: CacheClient;
```

Comment:
I'm assuming that this relies on the `CacheClient` itself implementing a TTL and that the default one will be used? Could be worth pointing that out in a doc comment here